angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/rating-review.html

```diff
@@ -0,0 +1,44 @@
+<md-dialog style='background: #fff;'>
+  <md-toolbar class='md-primary'>
+    <div class="md-toolbar-tools">
+      <h2>Rate {{::portlet.title}}</h2>
+      <span flex></span>
+      <md-button class="md-icon-button" ng-click="cancel()">
+        <i class='fa fa-times' aria-label="Close dialog"></i>
+      </md-button>
+    </div>
+  </md-toolbar>
+  <md-dialog-content>
+    <div class="md-dialog-content">
+      <div tabindex="0" id="Rated-{{::portlet.fname}}-description" aria-labelledby="Rated-{{::portlet.fname}}-description" class="help-block" ng-if="rating.previouslyRated">
```

Comment:
Shouldn't need an aria-label at all, since the content of the div provides enough information.